### DISCUSSION TITLE
Should VITE env variables trigger a cache miss?

### DISCUSSION THREAD

**Discussion by dcantu96** [vercel/turborepo#4855] [Help] [answered]
I use turbo on a react + vite monorepo that consists of 10+ react packages and 3 "demos" or vite projects... These vite projects have env vars like `VITE_API_URL`... and my question is:

Should turborepo trigger a cache miss on a vite build when the variables are different?

**Comment by mehulkar** [comment, marked as answer]
Yes, if a Vite app is detected, the values of env vars beginning with `VITE_` should be incorporated into the task hash. Changing the value of these env vars should cause cache misses. [See docs on framework inference here](https://turbo.build/repo/docs/core-concepts/caching/environment-variable-inputs#framework-inference) and file an issue with a minimal repro if something isn't working!
